src/java/htsjdk/variant/variantcontext/CommonInfo.java

```diff
@@ -90,38 +91,58 @@ public void setName(String name) {
     //
     // ---------------------------------------------------------------------------------------------------------
 
+    /** @return a <b>modifiable</b> Set of filters. Can be  null. All changes in this set will be reflected in the CommonInfo */
     public Set<String> getFiltersMaybeNull() {
         return filters;
     }
 
+    /** @return an unmodifiable Set of filters. Can be empty by never null */
     public Set<String> getFilters() {
         return filters == null ? NO_FILTERS : Collections.unmodifiableSet(filters);
     }
 
+    /** @return true if no filter has been defined  <code>(getFiltersMaybeNull()==null)</code> */
```

Comment:
It would be nice to note somewhere that in the parsing in the VCF codec if a site is unfiltered (represented by VCFConstants.UNFILTERED=".") then the filters list is set to null and that if the VC passes filters ("PASS"), its common info gets an empty Set<String>.